pkg/prometheus/promcfg.go

```diff
@@ -1511,6 +1511,26 @@ func (cg *ConfigGenerator) generateRemoteWriteConfig(version semver.Version, p *
 			cfg = append(cfg, yaml.MapItem{Key: "proxy_url", Value: spec.ProxyURL})
 		}
 
+		if spec.Sigv4 != nil {
```

Comment:
we need to check the Prometheus version since it's only supported for versions >= v2.26.0.